### PR TITLE
FHIR-39818,40108

### PR DESCRIPTION
**Pull request by abdullah-git1**

### CODE REVIEW

**Review by yanheras — CHANGES_REQUESTED** [review]
FHIR-39818:
1)	Change “Not Done Valueset” to “Not Done Value Set” to match the extension name. 
2) “The negation pattern for the MedicationRequest (MedicationNotRequested) resource is demonstrated …”: suggest rewording slightly to “The negation pattern for the MedicationRequest resource (QI Core Medication Not Requested) is demonstrated …”.

**Comment by yanheras** [comment]
Sorry to be picky. Could you change "Not Done Valueset" to "Not Done Value Set", value set as in two words? This would match the exact name of the extension, which is "Not Done Value Set". Thanks.

**Comment by abdullah-git1** [comment]
"Not Done Valueset" changed to  "Not Done Value Set"

**Review by yanheras — COMMENTED** [review]
Re: FHIR-40031
I merged this by mistake. I do have a few review comments with suggested changes:
1. Remove "MeasureReport" from the extension name and extension url. I think the extension name should be just location. This would be consistent with naming conventions used for other extensions added to MeasureReport. 
2. The description says "Location from which reported data was gathered. This extension is used to provide the location(s) from which reported data was gathered in a MeasureReport, both individual and summary measure report". This extension is added to the data exchange MeasureReport, but the definition only mentioned individual and summary.

**Review by yanheras — COMMENTED** [review]
Re: FHIR-40113
1) Update the short definition to "This id SHOULD reference population, stratifier, or supplementalData criteria id in a Measure resource."
 2) I think the context was right before, it didn't need to be changed
